app/assets/stylesheets/answers.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Answers controller here.
```

Comment:
It's a good idea to isolate the css for each controller, but even more so for each component that you create.